services/movie.py

```diff
@@ -6,22 +7,27 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: list[str] = None
```

Comment:
The `title` parameter should be a string, not a list. Change the type hint to `str` and adjust the logic accordingly.